R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2264,6 +2264,14 @@ test_that("createDataFrame sqlContext parameter backward compatibility", {
   expect_equal(collect(before), collect(after))
 })
 
+test_that("randomSplit", {
+  df <- createDataFrame(data.frame(id = 1:1000))
+  df_list <- randomSplit(df, c(2, 3, 5))
+  expect_equal(1000, sum(sapply(df_list, count)))
+  df_list <- randomSplit(df, c(2, 3, 5), 0)
+  expect_equal(1000, sum(sapply(df_list, count)))
```

Comment:
for the seeded version it will be good to add some test on how many elements should be present in each data frame (i.e. checking if the ratios are 0.2, 0.3 and 0.5 approximately)